Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/adapter/ContactDiaryOverviewNestedAdapter.kt

```diff
@@ -39,6 +39,10 @@ class ContactDiaryOverviewNestedAdapter(
             { key, _ ->
                 contactDiaryOverviewElementImage.setImageResource(key.drawableId)
                 contactDiaryOverviewElementName.text = key.text
+                contactDiaryOverviewElementName.contentDescription = if (key.type == ListItem.Type.LOCATION)
```

Comment:
Please use a `when` statement and list types explicitly (default is empty) so when we add additional types, they don't accidentally default to the peson type.

Reply:
I also think that using a `when` is better here.

Reply:
Good point, I will change it